src/Fantomas.Core/CodePrinter.fs

```diff
@@ -3110,7 +3110,7 @@ and genTypeDefn (TypeDef (ats, px, leadingKeyword, ao, tds, tcs, equalsRange, td
             (opt sepNone impCtor (fun mdf -> sepSpaceBeforeClassConstructor +> genMemberDefn mdf))
             (fun isMultiline ctx ->
                 if ctx.Config.AlternativeLongMemberDefinitions && isMultiline then
-                    genEqFixed SynTypeDefn_Equals equalsRange ctx
+                    sepSpace ctx |> genEqFixed SynTypeDefn_Equals equalsRange
```

Comment:
I would be a bit more explicit here.
You can check the presence of the implicit ctor alias:

```fsharp
let hasAliasInImplicitConstructor =
    match impCtor with
    | Some(SynMemberDefn.ImplicitCtor(selfIdentifier = Some _)) -> true
    | _ -> false
```

and then just tighten the condition:
```fsharp
if ctx.Config.AlternativeLongMemberDefinitions && isMultiline && not hasAliasInImplicitConstructor then
    genEqFixed SynTypeDefn_Equals equalsRange ctx
else
    genEq SynTypeDefn_Equals equalsRange ctx
```